mod-director-core/src/main/java/net/jan/moddirector/core/configuration/type/CurseRemoteMod.java

```diff
@@ -24,6 +26,8 @@
 public class CurseRemoteMod extends ModDirectorRemoteMod {
     private final int addonId;
     private final int fileId;
+    private final String folderName;
+    private String inject;
```

Comment:
Instead of adding these here, add them into `ModDirectorRemoteMod`, as they apply to all mod types.